compiler/test/input/sinister-tail-call-gc.gr

```diff
@@ -1,33 +0,0 @@
-module SinisterTailCallGC
-
-// Contrived example to test *ADVANCED* tail calls
```

Comment:
Deleted because it works now?

Reply:
Yeah, it's essentially the same as the other tests.